openapi-validator/README.md

```diff
@@ -0,0 +1,222 @@
+# RHOAS OpenAPI Validator
+
+A CLI for validating OpenAPI specifications against the RHOAS API Guidelines.
+
+## Development
+
+> NOTE: This project uses [Yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/) for easier development.
+
+Install dependencies:
+
+```shell
+yarn install
+```
+
+Build:
+
+```shell
+yarn build
+```
+
+**Running examples**
+
+Validate OpenAPI files using the uncompiled TypeScript CLI:
+
+```shell
+yarn validate-dev ./examples/openapi-valid.yaml
+```
+
+## Using
+
+It is recommended to use `npx` to validate your documents to ensure you use the latest validation rules:
+
+```shell
+npx @rhoas/openapi-validator validate ./path/to/openapi.yaml
+```
+
+## Rules
+
+The RHOAS ruleset extends the Spectral built-in "oas" ruleset (except `operation-tags`, `openapi-tags`). You can see the full list of rules from that ruleset [here](https://meta.stoplight.io/docs/spectral/docs/reference/openapi-rules.md)
+
+### oas3minimum
```

Comment:
Do you think it is worth to have fast/small docusaurus webpage that doesn't have front (always goes to the docs).